node/agent.go

```diff
@@ -59,6 +59,8 @@ import (
 const (
 	dockerEndpoint     = "unix:///var/run/docker.sock"
 	circularBufferSize = 1000
+	killQuiesce        = "quiesce=$(ps aux | grep '.*quiesce.*sh.*' | grep -v grep | awk '{print $2}');" +
```

Comment:
I still don't like this, because:
1. It only works if the quiesce/pause command contains the string "quiesce".  The command is defined in the service definition, so it could be anything.
2. attachAndRun could be used for other commands besides pausing, so we shouldn't kill "quiesce" anytime ANY command times out.

Can we succesfully grep the process out of ps using the passed-in `command` string?  If not, we need to figure something else out, like getting the pid when we run the command.